t/t0401-post-command-hook.sh

```diff
@@ -86,4 +86,44 @@ test_expect_success 'with post-index-change config' '
 	test_cmp expect post-command.out
 '
 
+test_expect_success 'skip hook for typos (no pre-command)' '
+	git init post-typo-no-pre &&
+	write_script post-typo-no-pre/.git/hooks/post-command <<-EOF &&
+	echo ran >post-command.out
+	EOF
+
+	GIT_TRACE2_PERF=1 git -C post-typo-no-pre version &&
```

Comment:
Oops. `GIT_TRACE2_PERF=1` here is a leftover from local debugging. Will fix after letting the PR builds complete.

Reply:
How about leaving it in? That could accelerate debugging in case something goes wrong in this test case in a CI build.